build/patches/Enable-crash-reporting-ui.patch

```diff
@@ -0,0 +1,872 @@
+-    Uploaded Crash Report ID:
++    Saved Crash Report File:
+   </message>
+   <message name="IDS_CRASH_REPORT_UPLOADED_TIME" desc="Label on chrome://crashes for the time at which the crash report was uploaded.">
+-    Upload Time:
++    Saved Time:
+   </message>
+   <message name="IDS_CRASH_REPORT_LOCAL_ID" desc="Label on chrome://crashes for the identifier of a crash report on the user's machine">
+     Local Crash Context:
+@@ -53,9 +53,15 @@
+     Crash reporting is disabled.
+   </message>
+   <message name="IDS_CRASH_UPLOAD_MESSAGE" desc="Link text for triggering crash uploading on chrome://crashes">
+-    Start uploading crashes
++    Start saving crashes
+   </message>
+   <message name="IDS_CRASH_UPLOAD_NOW_LINK_TEXT" desc="Link text for manual uploads of a crash report">
+-    Send now
++    Save now
++  </message>
++  <message name="IDS_CRASH_CLEAR_ALL_TEXT" desc="Link text for clear all crash files">
++    Clear all
++  </message>
++  <message name="IDS_CRASH_EXTRACT_NOW_TEXT" desc="Link text for manual generation of a crash report">
++    Generate now
+   </message>
+ </grit-part>
+diff --git a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/CrashFileManager.java b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/CrashFileManager.java
+--- a/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/CrashFileManager.java
++++ b/components/minidump_uploader/android/java/src/org/chromium/components/minidump_uploader/CrashFileManager.java
+@@ -107,6 +107,8 @@ public class CrashFileManager {
+ 
+     private static final Pattern TMP_PATTERN = Pattern.compile("\\.tmp\\z");
+ 
++    private static final String SAVED_MINIDUMP_ZIP_SUFFIX = ".zip";
++
+     // The maximum number of non-uploaded crashes that may be kept in the crash reports directory.
+     // Chosen to attempt to balance between keeping a generous number of crashes, and not using up
+     // too much filesystem storage space for obsolete crash reports.
+@@ -116,7 +118,7 @@ public class CrashFileManager {
+     // The maximum age, in days, considered acceptable for a crash report. Reports older than this
+     // age will be removed. The constant is chosen to be quite conservative, while still allowing
+     // users to eventually reclaim filesystem storage space from obsolete crash reports.
+-    private static final int MAX_CRASH_REPORT_AGE_IN_DAYS = 30;
++    private static final int MAX_CRASH_REPORT_AGE_IN_DAYS = 5;
+ 
+     // The maximum number of non-uploaded crashes to copy to the crash reports directory. The
+     // difference between this value and MAX_CRASH_REPORTS_TO_KEEP is that TO_KEEP is only checked
+@@ -460,9 +462,6 @@ public class CrashFileManager {
+     }
+ 
+     public void cleanOutAllNonFreshMinidumpFiles() {
+-        for (File f : getAllUploadedFiles()) {
```

Comment:
Do I understand correctly that the deletion is done elsewhere?

Reply:
fixed in https://github.com/bromite/bromite/pull/1105/commits/de5dacb811773f6d07423cec1e97e4689e85e7bc